### PR TITLE
Remove duplicate label definition

### PR DESCRIPTION
**Pull request by lecriste**
#### PR description:

This PR removes a duplication of label definitions. No changes are expected in the output.

#### PR validation:

`runTheMatrix -l limited`

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

File: SimDataFormats/Associations/python/LCToCPAssociation_cfi.py [hunk 1]
```diff
@@ -1,12 +1,8 @@
 import FWCore.ParameterSet.Config as cms
+from Validation.HGCalValidation.HGCalValidator_cfi import hgcalValidator
```
Comment (makortel) [inline SimDataFormats/Associations/python/LCToCPAssociation_cfi.py:2]: A DataFormat package should not depend on Validation package (even on python level). If a dependence between two packages is needed, the dependence should go in the other direction.

Written that, a DataFormat package should not define plugins (e.g. EDProducers). By quick look `SimCalorimetry/HGCalAssociatorProducers` could be a reasonable place for the producers.

Comment (lecriste) [reply SimDataFormats/Associations/python/LCToCPAssociation_cfi.py:2]: @makortel thanks for the comment.
By moving the plugin definition into
`SimCalorimetry/HGCalAssociatorProducers`, can I keep the dependency on Validation package there or shall I still swap the dependency direction?

Comment (makortel) [reply SimDataFormats/Associations/python/LCToCPAssociation_cfi.py:2]: I think also there the dependence should go by Validation depending on Simulation.

**Pull request merged**